opentreemap/importer/models/base.py

```diff
@@ -55,6 +60,10 @@ def row_count(self):
         return self.rows().count()
 
     def status_summary(self):
+        t = "Unknown Error During %s" if self.is_lost else "%s"
```

Comment:
If you change "During" to "While" the status message will read better:

Unknown Error During Not Yet Started
Unknown Error During Loading
Unknown Error During Preparing Verification
Unknown Error During Verifying
Unknown Error During Creating Trees

Unknown Error While Not Yet Started
Unknown Error While Loading
Unknown Error While Preparing Verification
Unknown Error While Verifying
Unknown Error While Creating Trees